src/views/app/folder-panel/lists-item/message-list-item.jsx

```diff
@@ -113,6 +113,7 @@ export default function MessageListItem({
 
 	const [showIcon, icon, iconTooltip, iconId, color] = useMemo(() => {
 		if (item) {
+			``;
```

Comment:
what is this for?

Reply:
I typo I think, thanks for spotting it out